index.js

```diff
@@ -11,28 +11,150 @@
-          //cell.style.background ="purple"; //with this propertie you can change the background color of the clicked cell. try comment the line bellow and uncomment this line. Do not forget to save this file and refresh the borwser to see the changes
-      });
-
-      li.appendChild(cell); //adding each cell into the row number x
-    }
+      for (var y = 0; y < 4; y++) {
+        const cell = document.createElement('div');
+        cell.className = "square"; // adding css properties to make it looks like a square
+        cell.textContent = `${x},${y}`;  // saves the coordinates as a string value 'x,y'
+        cell.value = 0;//state of the cell
+
+        //this function adds the click event to each cell
+        cell.addEventListener( 'click', playGame);
+            
+            //console.log( cell.textContent) //display the coordinates in the console
+            //cell.style.visibility = 'hidden';// this  means that the contents of the element will be invisible, but the element stays in its original position and size / try it clicking on any of the black cells (in your browser) and see whats happens
+            //cell.style.background ="purple"; //with this propertie you can change the background color of the clicked cell. try comment the line bellow and uncomment this line. Do not forget to save this file and refresh the borwser to see the changes
 
-     board_Player1.appendChild(li); //adding each row into the board
+        li.appendChild(cell); //adding each cell into the row number x
+      }
+     cells.appendChild(li); //adding each row into the board
+  }
 }
+
+let currentPlayer = player1;
+playerTurn.textContent = player1.name;
+
+// create function to start the game play
+function playGame(event) {
+  let cell = event.target; // get the element clicked
+  if (currentPlayer === player1 && board_Player2.contains(cell)) {
+    if (cell.value === 1 && cell.textContent !== 'ship') {
+      cell.style.background = 'green';
+      cell.textContent = 'ship';
+      player2_lives.textContent--;
+      if (player2_lives.textContent === "0") {
+        alert(`Congratulations ${currentPlayer.name}. You win!`);
+        return;
+      }
+      currentPlayer = player1;
+    } else if (cell.value === 0) {
+      cell.style.visibility = 'hidden';
+      currentPlayer = player2;
+      playerTurn.textContent = player2.name;
+    } 
+  } else if (currentPlayer === player2 && board_Player1.contains(cell)) {
+    if (cell.value === 1 && cell.textContent !== 'ship') {
+      cell.style.background = 'green';
+      cell.textContent = 'ship';
+      player1_lives.textContent--;
+      if (player1_lives.textContent === "0") {
+        alert(`Congratulations ${currentPlayer.name}. You win!`);
+        return;
+      }
+      currentPlayer = player2;
+    } else if (cell.value === 0) {
+      cell.style.visibility = 'hidden';
+      currentPlayer = player1;
+      playerTurn.textContent = player1.name;
+    }
+  }
```

Comment:
If  else statement starting in line 126 does exactly the same of if else starting in line 141. We can create a function that we can call twice with the proper arguments (player1 or player2).